### DISCUSSION TITLE
youtube download update error

### DISCUSSION THREAD

**Discussion by tack32** [smplayer-dev/smplayer#910] [General]
Some Youtube videos never start after I enter their URL. I thought to update but I get this message: "It's not possible to save C:/Users/abcd/.smplayer/yt-dlp.exe." I wonder why this is happening? Thanks

**Comment by smplayer-dev** [comment]
I don't know, maybe a permissions problem.
Anyway you can download yt-dlp.exe from here:
https://github.com/yt-dlp/yt-dlp/releases

**Reply by tack32** [reply]
OK. I'll give the download a try. Thanks!
